dclab/definitions.py

```diff
@@ -364,3 +365,52 @@ def get_feature_label(name, rtdc_ds=None):
 def scalar_feature_exists(name):
     """Convenience method wrapping `feature_exists(..., scalar_only=True)`"""
     return feature_exists(name, scalar_only=True)
+
+
+def update_dfn_with_feature_info(feature, label, is_scalar):
+    """Used by temporary features and plugin features to update the feature
```

Comment:
This and the two following functions can go anywhere, I thought here in the definitions file would be best. Should we make them protected i.e., `def _update_dfn_with_feature_info(...):` ?